spec/strava/api/client/endpoints/clubs/club_events_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'spec_helper'
+
+RSpec.describe 'Strava::Api::Client#club_events', vcr: { cassette_name: 'client/club_events' } do
+  include_context 'API client'
+  it 'returns club events' do
+    club_events = client.club_events('456773')
+    expect(club_events).to be_a Enumerable
```

Comment:
`expect(club_events.size).to eq ....` as well

Reply:
done